tests/unit/test_param_escaper.py

```diff
@@ -0,0 +1,108 @@
+from datetime import date, datetime
+import unittest, pytest
+
+from databricks.sql.utils import ParamEscaper, inject_parameters
+
+pe = ParamEscaper()
+
+class TestIndividualFormatters(object):
```

Comment:
do we need to add more tests for other supported types from here?
https://docs.databricks.com/sql/language-manual/sql-ref-datatypes.html

binary? struct? map?

Reply:
I don't think so, because this has to do with mapping user input into a SQL text query. Members of structs would be covered by a different type (string or number) (ditto for maps) and binary is represented as a string (already covered).